backend/index.js

```diff
@@ -10,6 +10,20 @@ app.get('/',(req,res)=>{
     res.send('Hello world');
 })
 
+app.get('/stockname/:NAME',(req, res)=>{
+    const headers = {headers:{'Authorization':process.env.QUOTE_API_KEY}}
+    let NAME = req.params.NAME
```

Comment:
One issue that I didnt know existed until I put the company name is that if the company name is lowercase, the API will not bring back any data. One thing you can do to insure that the name of the company is always upper case is to use the javascript [toUpperCase](https://www.w3schools.com/jsref/jsref_touppercase.asp) method

Reply:
Added the method and both lowercase and uppercase will display output.